Source/RequestBuilder.swift

```diff
@@ -481,10 +506,7 @@ import AdSupport
 
         if (uuidString == "") {
             let advertisingIdentifier: String = ASIdentifierManager.shared().advertisingIdentifier.uuidString
-
-            if (advertisingIdentifier != .kIFASentinelValue) {
```

Comment:
even if the device id is "0000" the value has to be passed in the jsonBody. removed the conditional checks